cpp/src/io/orc/reader_impl.cu

```diff
@@ -213,6 +214,19 @@ size_t gather_stream_info(const size_t stripe_index,
   return dst_offset;
 }
 
+/**
+ * @brief Determines if a column should be converted from decimal to float
+ */
+bool convert_decimal_column_to_float(const std::vector<std::string> &columns_to_convert,
```

Comment:
I think the name should be changed, current name suggests that the function performs the conversion.